src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/ParameterCapturingTests.cs

```diff
@@ -157,6 +157,9 @@ await RunTestCaseCore(
 
                 Assert.Equal(expectedCapturedMethod.TypeName, actualMethod.TypeName);
                 Assert.Equal(expectedCapturedMethod.MethodName, actualMethod.MethodName);
+                Assert.Equal(expectedCapturedMethod.ModuleName, actualMethod.ModuleName);
+                Assert.NotEqual(Guid.Empty, actualMethod.ModuleVersionId);
```

Comment:
We can calculate the actual expected values here. I believe the Exceptions functional tests already do this